src/containers/ThumbnailNavigation.js

```diff
@@ -25,7 +26,20 @@ const mapDispatchToProps = {
   setCanvas: actions.setCanvas,
 };
 
+/**
+ * Styles for withStyles HOC
+ */
+const styles = {
+  root: {
+    background: 'linear-gradient(to top, rgba(0,0,0,0.7) 0%, rgba(0,0,0,0.3) 70%, rgba(0,0,0,0) 100%)',
```

Comment:
Not a blocker in any way, but this makes me think we should figure out how we decide to style things; there are so many options! In components? As theme variables? In the stylesheet?

Reply:
(Love the gradient by the way)

Reply:
Totally agree, I'm still not 100% comfortable w/ deciding about where styles should go (but that may be a lot because css in js is still pretty foreign to me).